modified_files/net/net.gypi

```diff
@@ -1209,12 +1216,14 @@
       'quic/core/spdy_utils.cc',
       'quic/core/spdy_utils.h',
       'quic/platform/api/quic_aligned.h',
+#      'quic/platform/api/quic_bug_tracker.h',
```

Comment:
I thought all the quic related files should be uncommented here. The update script will by default comment newly added files.